src/app/assets/languages/en.json

```diff
@@ -135,7 +135,21 @@
       "templates-help" : "Create caches from a custom template.",
       "templates-placeholder" : "Select a cache template",
       "create-button-label": "Create",
-      "back-button-label": "Back"
+      "back-button-label": "Back",
+      "getting-started": {
+        "nav-title": "Get started",
+        "page-title": "Start with the basics",
+        "cache-name": "1. Enter a unique name for your cache.",
+        "cache-type": "2. Choose how to create your cache.",
+        "cache-type-radio1": "Create a new cache",
+        "cache-type-radio1-helper": "Build a cache with guided steps.",
+        "cache-type-radio2": "Add an existing cache",
```

Comment:
@karesti @oraNod I am not sure about this text, I think as we are creating a new cache by selecting this radio button. The text should be more related to new cache.

Reply:
@dipguptaredhat Yes, that's a good point. The intention was to say either build a cache from scratch via the wizard or supply an existing cache configuration from a file or some other source. But "existing cache" is not great. I've pushed a change just now.